Knock.net/Resources/Messages/MessagesResource.cs

```diff
@@ -23,7 +23,7 @@ public class MessagesResource : BaseResource
         /// <returns>A paginated Message response.</returns>
         public async Task<PaginatedResponse<Message>> List(Dictionary<string, object> options = null)
         {
-            if (options.ContainsKey("trigger_data"))
+            if (options != null && options.ContainsKey("trigger_data"))
```

Comment:
The default `options` value is null, so this was breaking when not passing an options Dictionary